n2y/config.py

```diff
@@ -127,6 +135,18 @@ def _validate_config_item(config_item):
     return True
 
 
+def _valid_filename_template(filename_template):
+    if not isinstance(filename_template, str):
+        logger.error("filename_template must be a string")
+        return False
+    try:
+        string.Formatter().parse(filename_template)
```

Comment:
Under what conditions will this fail/are you looking for?

Reply:
I think he meant to check that filename_template is a format string and I have adjusted this function to properly reflect that.